Scripts/GameManagement/GameManager.gd

```diff
@@ -33,22 +33,19 @@ remotesync func initialize_world():
 	if get_tree().is_network_server():
 		var zone = world.get_node("BattleRoyaleMap/Zone")
 		connect("game_started", zone, "_on_Game_Started")
+		
 
 remotesync func initialize_players():
 	print("initializing players...")
 	for p in LobbyManager.players:
 		var picked_class_scene = Globals.classes[LobbyManager.players[p]["class"]]["scene"]
 		var player = picked_class_scene.instance()
 		player.set_name(str(p))
-		var player_label = PlayerLabel.instance()
-		player_label.set_name("label_" + str(p))
-		player_label.get_node("Background/Name").text = LobbyManager.players[p]["name"]
 		world.add_child(player)
-		world.add_child(player_label)
 		player.position = random_spawnpoint()
-		player_label.position = player.position
 		players_info[p] = {"position": player.position, "body_rotation": player.get_node("Body").rotation}
 		get_tree().change_scene_to(world)
+	world.gameplay_info.rpc("create_labels")
```

Comment:
Create labels is remotesync. Function where it is called also. If there are 3 players, there will be 9 labels because everybody calls this. I suggest moving this somewhere else or changing functions logic.

Reply:
It was not remote before, but it was only called on the server then.